framework/decode/dx12_stats_consumer.h

```diff
@@ -31,7 +31,7 @@
 GFXRECON_BEGIN_NAMESPACE(gfxrecon)
 GFXRECON_BEGIN_NAMESPACE(decode)
 
-class Dx12StatsConsumer : public Dx12Consumer
+class Dx12StatsConsumer : public Dx12Consumer, public gfxrecon::decode::AnnotationHandler
```

Comment:
We don't need this multiple inheritance anymore.

Reply:
Done